src/helpers/gho-launch/README.md

```diff
@@ -0,0 +1,149 @@
+
+Guide on how to use the GHO Launch abstraction to launch GHO on a new chain. Let's say we have a chain called "Foo" and we want to launch GHO on it.
+
+## 1. Add the new chain to the CCIP constants
+
+Add the new chain corresponding values to the `CCIPChainRouters`, `CCIPChainSelectors`, `CCIPChainTokenAdminRegistries`, and `GhoCCIPChains` constant libraries.
+
+Do not forget to add the new chain to the `GhoCCIPChains`'s `getAllChains` function.
+
+```solidity
+/**
+ * @notice Returns all supported ChainInfo constants
+ * @return An array with all the ChainInfo constants supported
+ */
+function getAllChains() public pure returns (ChainInfo[] memory) {
+  ChainInfo[] memory allChains = new ChainInfo[](7);
+  allChains[0] = ETHEREUM();
+  allChains[1] = ARBITRUM();
+  allChains[2] = BASE();
+  allChains[3] = AVALANCHE();
+  allChains[4] = GNOSIS();
+  allChains[5] = INK();
+  allChains[6] = FOO(); // <-- The new chain
+  return allChains;
+}
+```
+
+## 2. Add remote lanes to existing CCIP pools
+
+To add a new remote lane between an existing CCIP pool and the pool on the new chain, you need to create a file inheriting from the `AaveV3GHOLane` base contract.
+
+For example, the proposal file to add a lane between the Ethereum CCIP pool and the Foo CCIP pool will look like this:
+
+```solidity
+contract Ethereum_Foo_AaveV3GHOLane_YYYYMMDD is AaveV3GHOLane {
+  constructor() AaveV3GHOLane(GhoCCIPChains.ETHEREUM()) {}
+
+  function lanesToAdd()
+    public
+    pure
+    override
+    returns (IUpgradeableBurnMintTokenPool_1_5_1.ChainUpdate[] memory)
+  {
+    return
+      _asArray(_asChainUpdateWithDefaultRateLimiterConfig(GhoCCIPChains.FOO()));
+  }
+}
+```
+
+You can override the `_defaultRateLimiterConfig` if a different rate limiter config is desired, or simply pass a different custom rate limiter config like this:
+
+```solidity
+contract Ethereum_Foo_AaveV3GHOLane_YYYYMMDD is AaveV3GHOLane {
+  constructor() AaveV3GHOLane(GhoCCIPChains.ETHEREUM()) {}
+
+  function lanesToAdd()
+    public
+    pure
+    override
+    returns (IUpgradeableBurnMintTokenPool_1_5_1.ChainUpdate[] memory)
+  {
+    return
+      _asArray(
+        _asChainUpdate(
+          GhoCCIPChains.FOO(),
+          IRateLimiter.Config({
+            isEnabled: true,
+            capacity: 1_000_000e18,
+            rate: 500
+          }),
+          IRateLimiter.Config({
+            isEnabled: true,
+            capacity: 500_000e18,
+            rate: 300
+          })
+        )
+      );
+  }
+}
+```
+
+### Testing of Remote GHO Lanes
+
+You can inherit `AaveV3GHORemoteLaneTest_PreExecution` and `AaveV3GHORemoteLaneTest_PostExecution` for pre and post proposal execution tests respectively, then override the required functions.
+
+For the case of the Ethereum remote lane, you need to inherit `AaveV3GHOEthereumRemoteLaneTest_PreExecution` and `AaveV3GHOEthereumRemoteLaneTest_PreExecution` instead, as they have special test cases for the Ethereum chain.
```

Comment:
did you mean post?

Reply:
Yes, good catch. Updated here: https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/d34bc3ff6855ea1899287f51826580499c67f3b0